client/app/account/billing/main/payments/details/billing-payments-details.html

```diff
@@ -1,96 +1,44 @@
-<div id="billing-payment-details">
+<div class="tab-content">
 
     <div data-ovh-alert></div>
 
-    <div class="page-header">
-        <h1>
-            <span data-translate="payment_details_title"></span>
-        </h1>
-    </div>
+    <oui-back-button data-ng-href="{{ ctrl.paymentsHref() }}"
+                     data-title="{{:: 'payment_details_return_to_payments' | translate }}">
+    </oui-back-button>
 
-    <div class="tab-content">
+    <dl class="dl-horizontal mt-4">
+        <dt data-translate="payment_details_payment_id"></dt>
+        <dd data-ng-bind="::ctrl.paymentId"></dd>
+        <dt data-translate="payment_details_payment_date"></dt>
+        <dd data-ng-bind="::ctrl.payment.date | date:'mediumDate'"></dd>
+        <dt data-translate="payment_details_payment_amount"></dt>
+        <dd data-ng-bind="::ctrl.payment.amount.text"></dd>
+    </dl>
 
-        <a href="#/billing/payments"
-           title="{{ 'payment_details_return_to_payments' | translate }}">
-            <i class="fa fa-chevron-left"
-               aria-hidden="true">
-            </i>
-            <span data-translate="payment_details_return_to_payments"></span>
-        </a>
+    <oui-datagrid data-rows-loader="ctrl.loadBills($config)"
+                  data-row-loader="ctrl.loadDetails($row)">
+        <oui-column data-title=":: 'payment_details_table_header_id' | translate"
+                    data-property="billId">
+        </oui-column>
+        <oui-column data-title=":: 'payment_details_table_header_date' | translate">
+            <span data-ng-bind="$row.date | date:'mediumDate'"></span>
+        </oui-column>
+        <oui-column data-title=":: 'payment_details_table_header_payment' | translate"
+                    data-property="payment">
+        </oui-column>
+        <oui-column data-title=":: 'payment_details_table_header_document' | translate">
+        </oui-column>
+        <oui-action-menu data-align="end"
+                         data-compact>
+            <oui-action-menu-item data-text="{{:: 'statements_details_table_invoice_html_link' | translate }}"
+                                  data-href="{{ $row.url }}"
+                                  data-external>
+            </oui-action-menu-item>
+            <oui-action-menu-item data-text="{{:: 'statements_details_table_invoice_pdf_link' | translate }}"
+                                  data-href="{{ $row.pdfUrl }}"
```

Comment:
No need for `::` ?

Reply:
i always prefer to avoid one-time binding dynamic data, but yeah it is safe to assume i could one time binding this one